spec/models/validations/sales/setup_validations_spec.rb

```diff
@@ -70,6 +74,10 @@
       context "when saledate is in the 22/23 collection year" do
```

Comment:
I think this wording might want updating.

Reply:
Yup!